app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -37,17 +37,17 @@
 
     private Retrofit retrofit;
 
-    private static SecurePreferences sSecurePreferences;
+    private static SecurePreferences securePreferences;
 
-    private static Context sContext;
+    private static Context context;
```

Comment:
Can we remove this variable (since it's available through `getApplicationContext()` anyway)?

Reply:
No we can't.
We need a static application context to pass to getSharedPreferences().
getApplicationContext() is a non-static one.

Reply:
Just saw your comment below, where you say that it's impossible. So ignore this.